### DISCUSSION TITLE
[Feature] image resize

### DISCUSSION THREAD

**Discussion by schmitzkr** [immich-app/immich#11457] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I find that I often need to resize images I have taken.  For example, I have a document or child's photo needed for an activity registration form, and the website or person I need to share it with will only accept specific file sizes.  My devices are often taking photos in the highest possible quality and so I am forced to reconfigure the device to shoot in lower quality or open another editor.
It would be great if the mobile editor could handle this.
This might be related to https://github.com/immich-app/immich/discussions/3865

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by iashraful** [comment]
This feature is really necessary. Please develop this.

**Comment by bo0tzz** [comment]
I consider this a dupe of #3865

**Reply by JeremyTBradshaw** [reply]
Hi @bo0tzz , I'm searching discussions but not finding another thread on this topic.  I like this feature request as-is, with no concerns about a download option.  I would just like to be able to have a job that will resize all  my original images.  Say find/replace all images of a given resolution, resize them all to a new resolution (locking aspect ratio, at least as an option).  Basically an in-place image resizer  to shrink the entire collection's file size.

Do you still see that as a dupe of the download related request?  I feel like I've read somewhere else that Immich team don't want to get into modifying the original image.  I can't seem to find that wherever it was. UPDATE: it's in the Docs page's FAQ where I read that Immich never changes the file (https://docs.immich.app/FAQ#is-it-possible-to-compress-images-during-backup / https://docs.immich.app/FAQ/#does-immich-change-the-file).  I guess that answers my question.

I really would like that feature.
